web-api/migration-terraform/main/lambdas/migrations/0003-update-trial-session-working-copy-status.js

```diff
@@ -0,0 +1,43 @@
+const isTrialSessionWorkingCopy = item => {
+  return item.pk.startsWith('trial-session-working-copy|');
+};
+
+const migrateItems = items => {
+  const itemsAfter = [];
+
+  for (const item of items) {
+    if (isTrialSessionWorkingCopy(item)) {
+      item.filters = {
+        basisReached: true,
+        continued: true,
+        definiteTrial: true,
+        dismissed: true,
+        motionToDismiss: true,
+        probableSettlement: true,
+        probableTrial: true,
+        recall: true,
+        rule122: true,
+        setForTrial: true,
+        settled: true,
+        showAll: true,
+        statusUnassigned: true,
+        submittedCAV: true,
+      };
+
+      for (const aCase in item.caseMetadata) {
+        switch (item.caseMetadata[aCase].trialStatus) {
+          case 'aBasisReached':
+            item.caseMetadata[aCase].trialStatus = 'basisReached';
+            break;
+          case 'takenUnderAdvisement':
+            item.caseMetadata[aCase].trialStatus = 'submittedCAV';
+            break;
+        }
+      }
+    }
```

Comment:
🔧 I think this migration should validate items it updates, we have a `validateWithLogging` method that I believe is typically used for migrations!

Reply:
Oh shoot, do we still have to validate in individual migrations in addition to running `0000-validate-all-items`?

Reply:
We don't necessarily have to! I am in favor because it shortens the feedback loop and also can help us debug when there is a validation error, especially if there are multiple migration scripts. My understanding of the `0000-validate-all-items` script up to this point was it is a safety measure to ensure we aren't migrating any invalid data but I could have misunderstood.

Reply:
Fair enough, thanks for clarifying! Adding it now